conf/common/system/example_slurm_cluster.toml

```diff
@@ -25,31 +25,29 @@ mpi = "pmix"
 gpus_per_node = 8
 ntasks_per_node = 8
 
-[partitions]
-  [partitions.partition_1]
-  name = "partition_1"
-  nodes = ["node-[001-100]"]
-
-  [partitions.partition_2]
-  name = "partition_2"
-  nodes = ["node-[101-200]"]
-
-    [partitions.partition_1.groups]
-      [partitions.partition_1.groups.group_1]
-      name = "group_1"
-      nodes = ["node-[001-025]"]
-
-      [partitions.partition_1.groups.group_2]
-      name = "group_2"
-      nodes = ["node-[026-050]"]
-
-      [partitions.partition_1.groups.group_3]
-      name = "group_3"
-      nodes = ["node-[051-075]"]
-
-      [partitions.partition_1.groups.group_4]
-      name = "group_4"
-      nodes = ["node-[076-100]"]
+[[partitions]]
```

Comment:
I agree with this change because it is a more TOML-native way to describe sections. However, we need to communicate with users to ensure they are aware of this change. @srinivas212, please review.

Reply:
@amaslenn, could you please create a PR for CloudAIX as well? The system configuration files will need to be updated accordingly once the PR is merged.